sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy

```diff
@@ -9,14 +9,7 @@ import com.azure.core.util.Context
 import com.azure.core.util.polling.SyncPoller
 import com.azure.storage.common.StorageSharedKeyCredential
 import com.azure.storage.common.implementation.Constants
-import com.azure.storage.file.share.models.NtfsFileAttributes
-import com.azure.storage.file.share.models.PermissionCopyModeType
-import com.azure.storage.file.share.models.ShareErrorCode
-import com.azure.storage.file.share.models.ShareFileCopyInfo
-import com.azure.storage.file.share.models.ShareFileHttpHeaders
-import com.azure.storage.file.share.models.ShareFileRange
-import com.azure.storage.file.share.models.ShareSnapshotInfo
-import com.azure.storage.file.share.models.ShareStorageException
+import com.azure.storage.file.share.models.*
```

Comment:
Let's not do `*` imports. https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-

Reply:
Weird, idk why but my settings there are at 100 for both but it's still wildcarding them. 

I've reverted this change.

Reply:
I usually set that one to 999.